pynicotine/gtkgui/search.py

```diff
@@ -99,8 +101,6 @@ def __init__(self, window):
         CompletionEntry(window.room_search_entry, window.room_search_combobox.get_model())
         CompletionEntry(window.search_entry, window.search_combobox.get_model())
 
-        self.file_properties = None
```

Comment:
This should remain

Reply:
Yes, good find!

Reply:
I think @mathiascode needs to approve our GitHub Actions workflows for your account before you can resolve review comments yourself. If a comment becomes outdated after a follow-up commit then it can be considered resolved.

Until then you can see the output of the linters and previous CI runs such as:
https://github.com/nicotine-plus/nicotine-plus/actions/runs/5169361075/jobs/9314244361

It is preferred to resolve review conversations but not mandatory. However, it is mandatory to resolve all linter (pylint and flake8) warnings before merging will be allowed.

Reply:
Yes, I did run the tests using pylint and flake8 on my local machine before checking in.
How do I trigger the tests in github for my last checkin?

Reply:
You can't at the moment, I don't think it's possible on the first PR to this repo. On my branch for https://github.com/catkryton/nicotine-plus/pull/1 the tests pass see https://github.com/slook/nicotine-plus/actions/runs/5186607952/jobs/9347936948